### PR TITLE
Remove patches

### PR DESCRIPTION
**Pull request by blindpirate**
Try to get rid of patches.

### CODE REVIEW

File: .teamcity/Gradle_Check/configurations/BaseGradleBuildType.kt [hunk 1]
```diff
@@ -15,5 +15,13 @@ open class BaseGradleBuildType(model: CIBuildModel, val stage: Stage? = null, us
 
     init {
         this.init()
+        cleanup {
```
Comment (blindpirate) [inline .teamcity/Gradle_Check/configurations/BaseGradleBuildType.kt:18]: @donat I'm not sure if it's correct to apply this cleanup rule to all build types. Why did you change specific build types to this cleanup rule?

Comment (donat) [reply .teamcity/Gradle_Check/configurations/BaseGradleBuildType.kt:18]: It's been a long time ago but I remember some builds produced a ridiculous amount of data and nobody volunteered to clean that up. Without the stricter clean up rules TeamCity disk would be filled up swiftly.

Comment (blindpirate) [reply .teamcity/Gradle_Check/configurations/BaseGradleBuildType.kt:18]: OK, let's keep this policy until someone complains.

**Comment by blindpirate** [comment]
@wolfs the time series was some graph TeamCity generates and displays. It was useless. 

I think 7 days artifact retention is probably acceptible.

**Comment by blindpirate** [comment]
@bot-gradle test QFL

File: .teamcity/Gradle_Check/configurations/BaseGradleBuildType.kt (hunk 1 above)
Comment (wolfs) [reply .teamcity/Gradle_Check/configurations/BaseGradleBuildType.kt:18]: What is the current retention? 14 days?

**Comment by blindpirate** [comment]
Thanks @wolfs @donat !

File: .teamcity/Gradle_Check/configurations/BaseGradleBuildType.kt (hunk 1 above)
Comment (blindpirate) [reply .teamcity/Gradle_Check/configurations/BaseGradleBuildType.kt:18]: I didn't find the official doc for this, but looking at the current history, it seems to be 30 days.

Comment (blindpirate) [reply .teamcity/Gradle_Check/configurations/BaseGradleBuildType.kt:18]: I can see the artifacts of May 29, but not May 26.

Comment (wolfs) [reply .teamcity/Gradle_Check/configurations/BaseGradleBuildType.kt:18]: Do we already keep the artifacts of failed builds for longer? If we do, then a retention of 7 days for successful builds seems fine.

Note that we seem to configure the default cleanup on the `Gradle` project, so if we really want to change retention, we should probably do it there or on the `Check` project, instead of configuring every project.

image
